packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx

```diff
@@ -117,24 +117,29 @@ const MedicationsDetailsTable: React.FC<ActiveMedicationsProps> = ({
             </p>
           </div>
           <p className={styles.bodyLong01}>
-            {medication.orderReasonNonCoded ? (
+            {medication.orderReasonNonCoded && (
               <span>
                 <span className={styles.label01}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
                 {medication.orderReasonNonCoded}
               </span>
-            ) : null}
-            {medication.quantity ? (
+            )}
+            {medication.quantity && (
               <span>
                 <span className={styles.label01}> &mdash; {t('quantity', 'Quantity').toUpperCase()}</span>{' '}
                 {medication.quantity} {medication?.quantityUnits?.display}
               </span>
-            ) : null}
-            {medication.dateStopped ? (
+            )}
+            {(medication.dateStopped || medication.autoStopDate) && (
               <span>
                 <span className={styles.label01}> &mdash; {t('endDate', 'End date').toUpperCase()}</span>{' '}
-                {formatDate(new Date(medication.dateStopped))}
+                {medication.dateStopped
+                  ? formatDate(new Date(medication.dateStopped))
+                  : formatDate(new Date(medication.autoStopDate))}
               </span>
-            ) : null}
+            )}
+            {(medication.dateStopped != null || medication.autoStopDate != null) && (
+              <span className={styles.label01}> &mdash; {t('discontinued', 'Discontinued').toUpperCase()}</span>
```

Comment:
Prefer to make the actual translation upper case instead of calling `toUpperCase()`.